setup/jest.json

```diff
@@ -1,8 +1,15 @@
 {
   "preset": "react-native",
   "moduleFileExtensions": ["ts", "tsx", "js"],
+  "globals": {
+    "ts-jest": {
+      "tsConfig": {
+        "isolatedModules": false
+      }
+    }
+  },
```

Comment:
as a followup, it feels like we should have this file generate a `jest.config.js` and `tsconfig.test.json` so we don't continue to bloat `package.json`

Reply:
https://github.com/echobind/react-native-template-echobind/issues/23